tfjs-backend-webgpu/BUILD.bazel

```diff
@@ -107,6 +107,9 @@ tfjs_web_test(
         '{"WEBGPU_CPU_FORWARD": false}',
     ],
     browsers = [],
-    local_browser = "chrome_webgpu",
+    local_browser = select({
+        "@bazel_tools//src/conditions:windows": "chrome_webgpu_windows",
```

Comment:
nit: Do we need to add the macOS?

Reply:
Good point. I renamed to Windows version to just `chrome_webgpu` and set it as the default condition. macOS will use it.